yfinance/scrapers/history.py

```diff
@@ -98,6 +99,14 @@ def history(self, period="1mo", interval="1d",
             if start is None:
                 if interval == "1m":
                     start = end - 604800  # Subtract 7 days
+                
+                elif interval == "1h":
```

Comment:
1h can fetch up to 730 days of data, subtracted 730 * 24 * 60 * 60 = 63,072,000 seconds